language/bytecode-verifier/src/dependencies.rs

```diff
@@ -342,74 +342,56 @@ impl<'a> DependencyChecker<'a> {
 pub struct CyclicModuleDependencyChecker {}
 
 impl CyclicModuleDependencyChecker {
-    pub fn verify_module<F: Fn(&ModuleId) -> Vec<ModuleId>>(
+    pub fn verify_module<F: Fn(&ModuleId) -> PartialVMResult<Vec<ModuleId>>>(
```

Comment:
Why does this return a result? That functionality doesn't seem to be used by the loader

And it seems particularly weird because the location of the error is likely NOT from the module you are verifying

Reply:
> Why does this return a result? 

I guess this is to cater to the possibility that the bytecode verifier is called outside the VM, and the result indicates the situation that hey we cannot load the module given this module id.

Reply:
I would remove this as its a bit bizarre. If the functionality is needed. we will add it later

Reply:
Done